lib/api/response_decoder.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:japx/japx.dart';
+
+Map<String, dynamic> decode(Map<String, dynamic> json) {
```

Comment:
I think we should wrap it into a class name ResponseDecodeder 🤔 
```suggestion
class ResponseDecoder {

static Map<String, dynamic> decode(Map<String, dynamic> json) {
```

Reply:
Thank you, I updated it at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/51/commits/7504b5f344ff3c450706889d59b1614edef186dc 🙌